third_party/cares/cares.BUILD

```diff
@@ -44,6 +44,36 @@ config_setting(
     values = {"cpu": "ios_arm64"},
 )
 
+config_setting(
```

Comment:
These additional CPU's are found in https://github.com/bazelbuild/bazel/blob/master/src/main/java/com/google/devtools/build/lib/rules/apple/ApplePlatform.java

Reply:
Putting this comment in the code (rather than github) would be more helpful I think :)